packages/protocol/contracts/governance/Governance.sol

```diff
@@ -838,6 +922,37 @@ contract Governance is IGovernance, Ownable, Initializable, ReentrancyGuard, Usi
     return voters[account].mostRecentReferendumProposal;
   }
 
+  /**
+   * @notice Checks if a byzantine quorum of validators has whitelisted the given hotfix.
+   * @param txHash The abi encoded keccak256 hash of the hotfix transaction.
+   * @return Whether validator whitelist tally >= validator byztanine quorum (2f+1)
+   */
+  function isHotfixPassing(bytes32 txHash) public view returns (bool) {
+    address[] memory validatorSet = getValidators().getRegisteredValidators();
```

Comment:
This isn't the actual validator set, instead, use the validatorSet precompile size